### PR TITLE
[Minor] Fix Java Lint errors introduced by #13286 and 13280

### PR DESCRIPTION
**Pull request by techaddict**
## What changes were proposed in this pull request?

Fix Java Lint errors introduced by #13286  and #13280
1. remove unused imports
2. Add exception for `Append` and `Complete` in lint-java
## How was this patch tested?

Existing tests

### CODE REVIEW

**Comment by srowen** [comment]
Oh, shouldn't we just fix that method name? I don't see why it was named `Append()` CC @tdas?

File: dev/checkstyle.xml [hunk 1]
```diff
@@ -157,7 +157,8 @@
             <property name="allowSamelineMultipleAnnotations" value="true"/>
         </module>
         <module name="MethodName">
-            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9_]*$"/>
+            <!-- Exclude Append and Complete were added in SPARK-15517 -->
+            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9_]*$|Append|Complete"/>
```
Comment (dongjoon-hyun) [inline dev/checkstyle.xml]: Hi, @techaddict .
Please use `dev/checkstyle-suppressions.xml` instead of changing rule.

**Comment by andrewor14** [comment]
This is not the right fix. Please close this PR.

**Comment by techaddict** [comment]
@andrewor14 btw making it lowercase(which was my first fix) or adding an
exception in check style suppression, please let me know what is the way
here

**Comment by andrewor14** [comment]
I believe it is intended to be upper case to match scala; it's just that for java it must be a method so that's failing the style. @srowen is there anyway to turn off the lint checks for a part of the code like `scalastyle:off`?

**Comment by techaddict** [comment]
@andrewor14 As @dongjoon-hyun has suggested simply added the file to `dev/checkstyle-suppressions.xml` should work

**Comment by srowen** [comment]
@andrewor14 ah OK if that was on purpose, this can just be suppressed.

**Comment by techaddict** [comment]
created a updated PR with couple of more java lint fixes #13559

**Pull request closed without merging**